file-formats/sito/examples/TM_SITO_TRANS_TEST_01.sito.json

```diff
@@ -0,0 +1,410 @@
+{
+  "formatVersion": "1",
+  "header": {
+    "description": "TM_SITO_TRANS_TEST_01",
+    "originalLanguage": "en",
+    "abapLanguageVersion": "cloudDevelopment"
+  },
+  "sit2Object": {
+    "sitnobjectid": "TM_SITO_TRANS_TEST_01",
+    "abapLanguageVersion": "5",
+    "sitnobjscope": "01",
```

Comment:
What does scope `01` mean? Can we introduce an enum so that the user gets meaningful names, here?